lib/solidus_dev_support/solidus_cli.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+require 'thor'
+require 'solidus_dev_support/extension'
+
+class SolidusDevSupport::SolidusCLI < Thor
```

Comment:
Couple of things:

1. Is there a reason for using `class SolidusDevSupport::SolidusCLI` instead of nesting `class SolidusCLI` within a `module SolidusDevSupport` definition?
2. `SoliudsCLI` is not compatible with Zeitwerk, which would expect it to be defined at `solidus_c_l_i.rb`. How about renaming it to `SolidusCli`? It's a no-brainer and prevents any potential problems.

Reply:
I do that by default because it keeps everything on a single level of indentation and avoids some constant lookup situations. But I'll expand it and rename the class to SolidusCommand. 

---

In the future I think it's something that should be extracted (to `solidus-cli`?) and added as a dependency of the `solidus` gem, but that's another story.